V1Documentation.md

```diff
@@ -1,53 +1,52 @@
-# Raygun4js
+# Raygun4js V1 Documentation
 
-[Raygun.io][raygunio] provider for client-side JavaScript
+The following documentation is for Raygun4js V1 only. Raygun4js V1 should only be used by legacy customers or for applications that block scripts.
 
-[raygunio]: https://raygun.io
+Go to [V2 documentation][v2docs].
+
+[v2docs]: https://github.com/MindscapeHQ/raygun4js/blob/master/README.md
 
 ## Getting Started
 
 **Step 1**
 
-Add this snippet to your markup, before the closing `</head>` tag:
+Download the [production version][min] or the [development version][max]. 
```

Comment:
The CDN should be the preferred option for V1, with the manual download links as an alternate option